civictechprojects/static/css/partials/_SubHeader.scss

```diff
@@ -199,7 +199,7 @@
   margin-left: 25px;
 }
 
-@media (min-width: 925px) {
+@media (min-width: 990px) {
```

Comment:
With the new font I think we need to break at 1000px

![image](https://user-images.githubusercontent.com/322346/59529173-1f6cb500-8e95-11e9-923f-02ce0f45b9c9.png)

Reply:
Which browser/OS is this, out of curiosity? I tested a few locally and found 985 was the widest breakpoint so I went to 990 for a buffer, clearly I needed to test a few more cases!

Reply:
Chrome/Win10